File: cleanup.js
const fs = require('fs');
const path = require('path');

function walkDir(dir, callback) {
  fs.readdirSync(dir).forEach(f => {
    let dirPath = path.join(dir, f);
    if (fs.statSync(dirPath).isDirectory()) {
      walkDir(dirPath, callback);
    } else {
      callback(dirPath);
    }
  });
}

const replacements = [
  { search: /"\/contact"/g, replace: '"/kontakt"' },
  { search: /'\/contact'/g, replace: "'/kontakt'" },
  { search: /#offerte-form/g, replace: '#angebot-form' },
  { search: /nav-offerte-cta/g, replace: 'nav-angebot-cta' },
  { search: /kenteken-schema/g, replace: 'kennzeichen-schema' },
  { search: /kenteken-bc-schema/g, replace: 'kennzeichen-bc-schema' },
  { search: /KentekenBestellenPage/g, replace: 'KennzeichenBestellenPage' },
  { search: /autosleutel/g, replace: 'autoschluessel' },
  { search: /bijmaken/g, replace: 'nachmachen' }
];

walkDir('./src', (filePath) => {
  if (filePath.endsWith('.tsx') || filePath.endsWith('.ts') || filePath.endsWith('.css') || filePath.endsWith('.md')) {
    let content = fs.readFileSync(filePath, 'utf8');
    let changed = false;
    
    replacements.forEach(r => {
      if (r.search.test(content)) {
        content = content.replace(r.search, r.replace);
        changed = true;
      }
    });

    if (changed) {
      fs.writeFileSync(filePath, content);
      console.log('Updated ' + filePath);
    }
  }
});

File: cleanup.py
import os
import re

replacements = [
    (r'"/contact"', '"/kontakt"'),
    (r"'/contact'", "'/kontakt'"),
    (r'#offerte-form', '#angebot-form'),
    (r'nav-offerte-cta', 'nav-angebot-cta'),
    (r'kenteken-schema', 'kennzeichen-schema'),
    (r'kenteken-bc-schema', 'kennzeichen-bc-schema'),
    (r'KentekenBestellenPage', 'KennzeichenBestellenPage'),
    (r'autosleutel', 'autoschluessel'),
    (r'bijmaken', 'nachmachen')
]

for root, dirs, files in os.walk('./src'):
    for file in files:
        if file.endswith('.tsx') or file.endswith('.ts') or file.endswith('.css') or file.endswith('.md'):
            filepath = os.path.join(root, file)
            with open(filepath, 'r', encoding='utf-8') as f:
                content = f.read()
            
            original = content
            for search, replace in replacements:
                content = re.sub(search, replace, content)
            
            if content != original:
                with open(filepath, 'w', encoding='utf-8') as f:
                    f.write(content)
                print(f"Updated {filepath}")
